lib/workers/global/config/parse/file.ts

```diff
@@ -18,9 +18,13 @@ export function getConfig(env: NodeJS.ProcessEnv): AllConfig {
     if (err instanceof SyntaxError || err instanceof TypeError) {
       logger.fatal(`Could not parse config file \n ${err.stack}`);
       process.exit(1);
+    } else if (env.RENOVATE_CONFIG_FILE) {
+      logger.fatal('No custom config file found on disk');
+      process.exit(1);
```

Comment:
@rarkins Shouldn't we throw a config error instead? 🤔

Reply:
I don't mind. Exiting immediately is consistent with the earlier if too

Reply:
Do let me know your preference and I can help to make the change. Thanks!

Reply:
Let's keep it as is